inscriptions/ISic000571.xml

```diff
@@ -185,36 +190,40 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la">
+            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR138359" xml:lang="la" resp="#GC">
                 <ab>
                     <lb n="1"/>
                     <expan>
                         <abbr>D</abbr>
                         <ex>is</ex>
-                    </expan>
+                    </expan> <g ref="#interpunct">·</g>
                     <expan>
-                        <abbr>M</abbr>
+                        <abbr>m</abbr>
                         <ex>anibus</ex>
                     </expan>
-                    .<lb n="2"/>Suetia Tyndaris
+                    .<lb n="2"/><name>Suetia</name> <g ref="#interpunct">·</g> Tyndaris <!--Not sure whether she's from Tyndaris or if this is part of her name?-->
```

Comment:
Fair question. One must assume it's a personal name (tyndaritana or similar would be the ethnic), but there are obvious reasons why she might have taken the name / been given it.